modular_skyrat/master_files/code/game/objects/structures/tables_racks.dm

```diff
@@ -4,49 +4,17 @@
 		return
 	return attack_hand(user, modifiers)
 
-//Most of this is code that allows stasis and numbing to work on surgery tables.
-/obj/structure/table/optable
-	/// Is the table able to put patients in stasis?
-	var/stasis_capable = FALSE
-	/// Is the Operating table able to preform numbing?
-	var/numbing_capable = TRUE
```

Comment:
The removal of the stasis mode is not mentioned anywhere inside of your PR, please document this change.

Reply:
Stasis mode is not used anywhere.